pages/science/flames.rst

```diff
@@ -125,6 +125,40 @@ where :math:`D_{ki}` is the multicomponent diffusion coefficient and
 :math:`D_k^T` is the Soret diffusion coefficient (used only if calculation of
 this term is specifically enabled).
 
+The Drift-Diffusion Model
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To account for the transport of charged species in a flame, the drift term is added to the
+diffusive fluxes of the mixture-average formulation according to [Ped1993]_,
+
+.. math::
+  
+   j_k^* = \rho \frac{W_k}{\overline{W}} D_{km}^\prime \frac{\partial X_k}{\partial z} +
+           s_k \mu_k E Y_k,
+
+where :math:`s_k` is the sign of charge (1,-1, and 0 respectively for positive, negative, and
+neutral charge), :math:`\mu_k` is the mobility, and :math:`E` is the electric field.
+The diffusion coefficients and mobilities of charged species can be more accurately calculated by
+**IonGasTransport::getMixDiffCoeffs** and **IonGasTransport::getMobilities**. In addition, the
+correction can only be applied to the fluxes of neutral species to ensure the accuracy of the
+fluxed of charged species,
+
+.. math::
+
+    j_k = j_k^* - \frac {1 - |s_k|} {1 - \sum_i |s_i| Y_i} Y_k \sum_i j_i^*.
+
+The equation for electrostatics is added to obtain the electric field,
```

Comment:
The equation is added _where_?

Currently, this is mixed in with the discussion of diffusion terms. If you want to fully integrate the description of ionized flows with the "normal" flame model, I think this would need to be introduced earlier, alongside the species / energy equations.

The other option would be to have a specific section after everything about normal flames (that is, after all the description of all the boundary conditions) to introduce the additional considerations for ionized flames. I don't have a strong preference one way or the other.